src/map/skill.cpp

```diff
@@ -16937,9 +16937,11 @@ struct s_skill_condition skill_get_requirement(struct map_session_data* sd, uint
 		case SO_FIRE_INSIGNIA:
 		case SO_WIND_INSIGNIA:
 		case SO_EARTH_INSIGNIA:
-		case WZ_FIREPILLAR: // no gems required at level 1-5 [celest]
 			req.itemid[0] = skill->require.itemid[min(skill_lv-1,MAX_SKILL_ITEM_REQUIRE-1)];
```

Comment:
judging from this, we could set different itemid per lvl, did we break something or ?
can't our skilldb.yaml support this ?